pallets/funding/src/types.rs

```diff
@@ -157,47 +157,77 @@ pub mod config_types {
 }
 
 pub mod storage_types {
-	use sp_arithmetic::traits::{One, Saturating, Zero};
+	use itertools::Itertools;
+	use sp_arithmetic::{
+		traits::{One, Saturating, Zero},
+		Percent,
+	};
+	use sp_std::collections::btree_set::BTreeSet;
 
 	use super::*;
 
-	#[derive(Default, Clone, Encode, Decode, Eq, PartialEq, RuntimeDebug, MaxEncodedLen, TypeInfo)]
+	#[derive(Clone, Encode, Decode, Eq, PartialEq, RuntimeDebug, MaxEncodedLen, TypeInfo)]
 	#[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
-	pub struct ProjectMetadata<BoundedString, Balance: BalanceT, Price: FixedPointNumber, AccountId, Hash> {
+	pub struct ProjectMetadata<
+		BoundedString,
+		Balance: BalanceT,
+		Price: FixedPointNumber,
+		AccountId,
+		Hash,
+		RoundTicketSizes: TicketSizeValidityCheck<Price>,
+	> {
 		/// Token Metadata
 		pub token_information: CurrencyMetadata<BoundedString>,
 		/// Mainnet Token Max Supply
 		pub mainnet_token_max_supply: Balance,
 		/// Total allocation of Contribution Tokens available for the Funding Round. (Auction, Community)
```

Comment:
We can now remove/rephrase the `(Auction, Community)` tuple from the comment.

Reply:
I'm actually referring to

`/// Total allocation of Contribution Tokens available for the Funding Round. (Auction, Community)`